### PR TITLE
Create GitHub actions

### PR DESCRIPTION
**Pull request by Pchelolo**
Testing GH actions for CI

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+on: [push,pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [6.x, 10.x]
```
Comment (jdforrester) [inline .github/workflows/nodejs.yml]: Should probably initiate testing on 12.x at some point.

**Comment by jdforrester** [comment]
This is quite the set of tests!

**Comment by Pchelolo** [comment]
> This is quite the set of tests!

hehe, it's mostly a version/configuration matrix.. If only I could make it work...

btw, I'm impressed with GH actions, I've spent no more then 15 minutes on it and it's almost at a working state. Very nice.

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -1,12 +1,16 @@
 name: Node.js CI
 
-on: [push,pull_request]
+on:
```
Comment (jdforrester) [inline .github/workflows/nodejs.yml]: You can simplify this down to just `on: [push,pull_request]` unless non-master patches happen a lot?

Comment (Pchelolo) [reply .github/workflows/nodejs.yml]: In that case it runs both 'push' and 'pull_request' if the development branch is in upstream and not in the fork as it is the case with this one. We usually don't do it, this time it happened cause I've used GitHub UI to create the actions file, but being explicit wouldn't hurt imho

**Comment by Pchelolo** [comment]
Hmmm... I don't see a way to only rerun failed jobs.. Travis had that. But otherwise I think it's neat, we can live with it.

**Pull request merged**